Bhagyashri/first_assignment_17-july-2019/css/style.css

```diff
@@ -0,0 +1,114 @@
+
+body{
+background-color:#71D0C5;
+padding:0 100px;
```

Comment:
Here it is better that you use padding in percentage. 
As we need to cover the other devices as well.